Tools/wasm/README.md

```diff
@@ -281,6 +286,50 @@ make -j$(nproc)
 popd
 ```
 
+## Running Python in WASI
+
+Here is a simple script that can run a python script, it expects that the Python installation
+is at the root of the filesystem (and it appears that currently that is [required](https://github.com/python/cpython/issues/100307).
+
+```shell
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+    FILE=""
+    HOST_DIR=$PWD
+    GUEST_DIR=$PWD
+elif [[ "$1" = /* ]]; then
+    REAL_PATH=$(realpath $1)
+    HOST_DIR=$(dirname $REAL_PATH)
+    GUEST_DIR=$HOST_DIR
+    FILE=${REAL_PATH}
```

Comment:
Maybe to help make things clearer around the difference between host and guest directories it's worth mapping to a `/workspace` guest directory?